frontend/alert/src/components/managealert/AlertItem.js

```diff
@@ -21,7 +21,7 @@ const StatusGridItem = styled(GridItem)`
 
 // Component for an alert entry (renders as a row in CSS grid)
 export const AlertItem = ({
-    date, course, status, repeat, actions, rownum,
+    date, course, status, repeat, actions, rownum, checked, toggleAlert, actionHandler,
```

Comment:
would rename `actionHandler` to something more specific -- what action is this for? When is it invoked?